azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/tools/auth/model/AuthType.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.tools.auth.model;
+
+/**
+ * The auth type which user may define in configuration.
+ */
+public enum AuthType {
+    AUTO,
+    AZURE_CLI,
+    MANAGED_IDENTITY,
```

Comment:
`MANAGED_IDENTITY` refers to Service principal?

Reply:
No, it is a new auth in azure resources

Reply:
![image](https://user-images.githubusercontent.com/17075863/103497840-b5a06b80-4e7d-11eb-90e2-bc76c8adc295.png)